tests/test_modelgen.py

```diff
@@ -3,7 +3,7 @@
 import pytest
 from tensorflow.keras.models import Model
 from mcfly import modelgen
-from mcfly.models import CNN, ConvLSTM, ResNet, InceptionTime
+from mcfly.models import ResNet
```

Comment:
This seems like an unused import.
```suggestion
```

Reply:
It is not, it is used in test test_generate_models_pass_model_object